lib/npg_qc/Schema/Result/TagMetrics.pm

```diff
@@ -188,6 +188,24 @@ A foreign key referencing the id_seq_composition column of the seq_composition t
   data_type: 'bigint'
   is_nullable: 1
 
+=head2 pct_tag_hops
```

Comment:
This would break a real deployment as this column would not exist - can you add tests to catch errors of this kind (in a separate later PR if need be) please?